.travis.yml

```diff
@@ -18,6 +18,8 @@ before_script:
 
 script:
   - ng test --reporters=spec --browser=ChromeHeadless --single-run=true
+  # this should really be fixed in the package itself
+  - cp fix/jdnconvertiblecalendardateadapter/index.d.ts node_modules/jdnconvertiblecalendardateadapter/dist/index.d.ts
```

Comment:
once we use the angular compiler to build `jdnconvertiblecalendardateadapter`, this should not be necessary any more. See https://github.com/dhlab-basel/JDNConvertibleCalendarDateAdapter/issues/5